### ISSUE TITLE
Fatal signal 6 (SIGABRT) at 0x0000032e (code=0), thread 20791 (robot_for_phone)

### ISSUE THREAD

**Issue by btelman96** [btelman96/runmyrobot_android#17]
Device: ZTE Speed
OS: 4.4.4

```
> 09-09 21:04:59.922 814-828/? E/ActivityManager: ANR in com.runmyrobot.android_robot_for_phone (com.runmyrobot.android_robot_for_phone/.activities.MainRobotActivity)
>     PID: 20791
>     Reason: Input dispatching timed out (Waiting because no window has focus but there is a focused application that may eventually add a window when it finishes starting up.)
>     Load: 5.66 / 4.99 / 5.22
>     CPU usage from 3846ms to -4212ms ago:
>     26% 814/system_server: 11% user + 15% kernel / faults: 1970 minor 33 major
>     0% 252/debuggerd: 0% user + 0% kernel / faults: 2679 minor 26 major
>     4.6% 1128/com.google.android.gms.persistent: 2.8% user + 1.7% kernel / faults: 991 minor 15 major
>     4.1% 1238/com.google.android.googlequicksearchbox: 1.7% user + 2.3% kernel / faults: 1734 minor 25 major
>     1.6% 158/mmcqd/0: 0% user + 1.6% kernel
>     1.6% 1149/com.android.phone: 0.6% user + 0.9% kernel / faults: 548 minor 33 major
>     1.2% 96/kswapd0: 0% user + 1.2% kernel
>     1.1% 403/adbd: 0% user + 1.1% kernel
>     1.1% 967/com.android.systemui: 0.4% user + 0.6% kernel / faults: 329 minor 4 major
>     0.9% 256/surfaceflinger: 0.1% user + 0.8% kernel / faults: 92 minor 11 major
>     0.3% 1099/com.google.android.inputmethod.latin: 0.1% user + 0.2% kernel / faults: 627 minor 20 major
>     0.1% 1182/com.qualcomm.services.location: 0% user + 0.1% kernel / faults: 508 minor 25 major
>     0.8% 20791/com.runmyrobot.android_robot_for_phone: 0.4% user + 0.3% kernel / faults: 153 minor 5 major
>     0.7% 1569/mpdecision: 0% user + 0.7% kernel / faults: 1 major
>     0.7% 19742/vmstat_update+0: 0% user + 0.7% kernel
>     0% 1165/com.redbend.vdmc: 0% user + 0% kernel / faults: 937 minor 19 major
>     0.1% 1198/com.android.modem.service: 0% user + 0% kernel / faults: 472 minor 15 major
>     0.6% 19918/flush_to_ldisc+: 0% user + 0.6% kernel
>     0% 1212/com.zte.extension: 0% user + 0% kernel / faults: 365 minor 12 major
>     0% 1224/com.skyfire.browser.toolbar.service: 0% user + 0% kernel / faults: 389 minor 12 major
>     0.4% 9686/cfq_kick_queue+: 0% user + 0.4% kernel
>     0.4% 20145/async_run_entry: 0% user + 0.4% kernel
>     0.3% 3/ksoftirqd/0: 0% user + 0.3% kernel
>     0.3% 259/mediaserver: 0% user + 0.3% kernel / faults: 56 minor 59 major
>     0.3% 19813/logcat: 0% user + 0.3% kernel
>     0% 7788/delayed_fput+0x: 0% user + 0% kernel
>     0.2% 9845/kworker/2:3: 0% user + 0.2% kernel
>     0.2% 13836/VosRXThread: 0% user + 0.2% kernel
>     0.2% 19297/def_work_fn+0x0: 0% user + 0.2% kernel
>     0% 7/migration/0: 0% user + 0% kernel
>     0.1% 8/rcu_preempt: 0% user + 0.1% kernel
>     0.1% 14/ksoftirqd/1: 0% user + 0.1% kernel
>     0% 18/migration/2: 0% user + 0% kernel
>     0.1% 37/msm_rpm_smd_wor: 0% user + 0.1% kernel
>     0.1% 189/jbd2/mmcblk0p32: 0% user + 0.1% kernel
>     0.1% 506/mdss_fb0: 0% user + 0.1% kernel
>     0.1% 8667/kworker/3:3: 0% user + 0.1% kernel
>     0.1% 18035/_kgsl_event_wor: 0% user + 0.1% kernel
>     0.1% 19859/com.google.android.talk: 0% user + 0.1% kernel / faults: 175 minor 3 major
>     0% 20977/kworker/0:2H: 0% user + 0% kernel
>     1.8% TOTAL: 0% user + 0% kernel + 0% softirq
>     
>     CPU usage all the time {uptime user system iowait Txbyte Rxbyte}
>     added by jiangmin 2014.06
>     8.4%    { 15% 1254449 68254 37595 0 0 0 } 284 : 1006/ mm-qcamera-daemon: 5.4% user + 2.9% kernel / faults: 4828948 minor 927 major
>     3.2%    { 6% 1277390 20931 21031 0 743568 823496 } 814 : 0/ system_server: 1.6% user + 1.6% kernel / faults: 782904 minor 3118 major
>     3%    { 5.6% 1277391 10021 29366 0 7614 11209 } 256 : 1000/ surfaceflinger: 0.7% user + 2.2% kernel / faults: 40666 minor 119 major
>     2.4%    { 4.5% 1273357 2357 29407 0 743568 823496 } 403 : 0/ adbd: 0.1% user + 2.3% kernel / faults: 2940 minor 6 major
>     2.3%    { 4.3% 1277391 8574 21630 0 0 0 } 259 : 1013/ mediaserver: 0.6% user + 1.6% kernel / faults: 156303 minor 484 major
>     0.9%    { 1.6% 1275448 6685 5104 0 743568 823496 } 1640 : 0/ com.google.android.gms: 0.5% user + 0.4% kernel / faults: 658195 minor 2590 major
>     0.7%    { 1.2% 1253894 8041 935 0 743568 823496 } 3985 : 0/ com.google.android.tts: 0.6% user + 0% kernel / faults: 314219 minor 1328 major
>     0.6%    { 1.1% 1276123 4768 3435 0 743568 823496 } 1128 : 0/ com.google.android.gms.persistent: 0.3% user + 0.2% kernel / faults: 245134 minor 1443 major
>     0.4%    { 0.8% 1276123 35
> 09-09 21:04:59.932 259-20989/? D/ExtendedUtils: extended extractor not needed, return default
> 09-09 21:04:59.932 259-20989/? D/AudioPolicyManager: copl: isOffloadSupported: SR=48000, CM=0x1, Format=0x7000000, StreamType=7, BitRate=4294967295, duration=418187 us, has_video=0
>     copl: isOffloadSupported: stream_type != MUSIC, returning false
> 09-09 21:04:59.932 259-20989/? D/OMXCodec: Successfully allocated OMX node 'OMX.google.vorbis.decoder'
> 09-09 21:04:59.932 814-828/? I/Process: Sending signal. PID: 20791 SIG: 6
> 09-09 21:04:59.932 259-20989/? I/AwesomePlayer: Could not offload audio decode, try pcm offload
> 09-09 21:04:59.932 20791-20791/com.runmyrobot.android_robot_for_phone A/libc: Fatal signal 6 (SIGABRT) at 0x0000032e (code=0), thread 20791 (robot_for_phone)
> 09-09 21:04:59.932 259-20989/? D/AudioPolicyManager: copl: isOffloadSupported: SR=44100, CM=0x1, Format=0x17000001, StreamType=7, BitRate=4294967295, duration=418187 us, has_video=0
>     copl: isOffloadSupported: stream_type != MUSIC, returning false
> 09-09 21:04:59.932 814-4498/? D/PowerManagerService: sendPendingNotificationsLocked mSendGoToSleepFinishedNotificationWhenReady = false
> 09-09 21:04:59.932 814-2581/? D/PowerManagerService: sendPendingNotificationsLocked mSendGoToSleepFinishedNotificationWhenReady = false
> 09-09 21:04:59.942 259-961/? I/ExtendedUtils: AV Sync late margin : Intended=40ms Using=0ms
> 09-09 21:04:59.942 259-961/? D/Utils: printFileName /system/media/audio/ui/VideoRecord.ogg
> 09-09 21:04:59.942 259-961/? I/AwesomePlayer: setDataSource_l(URL suppressed)
> 09-09 21:04:59.952 258-509/? D/PRDrmPlugIn: PRDrmPlugin::onInitialize, uniqueId = 7898
>     PRDrmPlugin::onSetOnInfoListener: starts! uniqueId = 7898
>     PRDrmPlugin::onSetOnInfoListener: ends!
>     PRDrmPlugin::onTerminate, uniqueId = 7898
> 09-09 21:04:59.962 259-20993/? D/ExtendedUtils: extended extractor not needed, return default
> 09-09 21:04:59.962 259-20993/? D/AudioPolicyManager: copl: isOffloadSupported: SR=48000, CM=0x1, Format=0x7000000, StreamType=7, BitRate=4294967295, duration=543333 us, has_video=0
>     copl: isOffloadSupported: stream_type != MUSIC, returning false
> 09-09 21:04:59.972 259-20993/? D/OMXCodec: Successfully allocated OMX node 'OMX.google.vorbis.decoder'
> 09-09 21:04:59.972 259-20993/? I/AwesomePlayer: Could not offload audio decode, try pcm offload
> 09-09 21:04:59.972 259-20993/? D/AudioPolicyManager: copl: isOffloadSupported: SR=44100, CM=0x1, Format=0x17000001, StreamType=7, BitRate=4294967295, duration=543333 us, has_video=0
>     copl: isOffloadSupported: stream_type != MUSIC, returning false
> 09-09 21:05:00.042 814-20990/? D/dalvikvm: GC_FOR_ALLOC freed 629K, 17% free 16584K/19960K, paused 94ms, total 95ms
> 09-09 21:05:00.052 814-20990/? I/dalvikvm-heap: Grow heap (frag case) to 18.808MB for 524304-byte allocation
> 09-09 21:05:00.052 259-961/? I/CameraClient: Opening camera 0
> 09-09 21:05:00.052 252-252/? I/DEBUG: *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
>     Build fingerprint: 'zte/N9130/speed:4.4.4/KTU84P/20150828.161233.30459:user/release-keys'
>     Revision: '0'
> 09-09 21:05:00.052 259-961/? I/QCamera2HWI: static void qcamera::QCamera2HardwareInterface::getLogLevel() gCamHalLogLevel=1
> 09-09 21:05:00.052 259-20998/? D/QCameraStateMachine: static void* qcamera::QCameraStateMachine::smEvtProcRoutine(void*): E
> 09-09 21:05:00.052 252-252/? I/DEBUG: pid: 20791, tid: 20791, name: robot_for_phone  >>> com.runmyrobot.android_robot_for_phone <<<
>     signal 6 (SIGABRT), code -6 (SI_TKILL), fault addr --------
> 09-09 21:05:00.052 259-961/? D/QCamera2HWI: [KPI Perf] int qcamera::QCamera2HardwareInterface::openCamera(hw_device_t**): E PROFILE_OPEN_CAMERA camera id 0
> 09-09 21:05:00.052 259-961/? E/mm-camera-intf: mm_camera_open: dev name = /dev/video1, cam_idx = 1
> 09-09 21:05:00.052 259-20999/? D/QCamera2HWI: static void* qcamera::QCamera2HardwareInterface::defferedWorkRoutine(void*): start data proc
> 09-09 21:05:00.052 284-21000/? E/mm-camera: mct_pipeline_start_session_thread thread_id is 21000
> 09-09 21:05:00.052 284-21000/? E/mm-camera-sensor: module_sensor_start_session:710 session 1
> 09-09 21:05:00.052 284-21001/? E/mm-camera: mct_pipeline_start_session_thread thread_id is 21001
> 09-09 21:05:00.052 284-21002/? E/mm-camera: mct_pipeline_start_session_thread thread_id is 21002
> 09-09 21:05:00.052 814-932/? W/AlarmManager: Alarm triggering: Alarm{42c7bb50 type 3 android}
> 09-09 21:05:00.072 284-21002/? E/mm-camera: isp_tintless_open: E: b6e3366d
> 09-09 21:05:00.072 284-21003/? E/mm-camera: mct_pipeline_start_session_thread thread_id is 21003
> 09-09 21:05:00.072 284-21000/? E/mm-camera-sensor: sensor_set_position:1401 Position ID: 0
>     sensor_set_mode:1432 Sensor Mode ID: 0
>     eeprom_load_library:535 eeprom_load_library lib_name libmmcamera_common_ar0542_eeprom.so
> 09-09 21:05:00.072 284-21002/? E/mm-camera: isp_tintless_get_version: lib returned version 3.61 err=0
> 09-09 21:05:00.072 284-21006/? E/mm-camera: isp_thread_session_task thread_id is 21006
> 09-09 21:05:00.072 284-21004/? E/mm-camera: mct_pipeline_start_session_thread thread_id is 21004
> 09-09 21:05:00.072 284-21004/? D/mm-camera: cpp_module_start_session:344, info: starting session 1
> 09-09 21:05:00.072 284-21005/? E/mm-camera: isp_tintless_main_loop thread_id is 21005
> 09-09 21:05:00.072 284-21000/? E/mm-camera-sensor: eeprom_load_library:556 e_ctrl->eeprom_lib.func_tbl =0xb59ae004
> 09-09 21:05:00.082 284-21007/? E/mm-camera: mct_pipeline_start_session_thread thread_id is 21007
> 09-09 21:05:00.082 284-21007/? E/hdr: Session id 0x1 started
> 09-09 21:05:00.082 284-21009/? E/mm-camera-img: module_imglib_msg_thread thread_id is 21009
> 09-09 21:05:00.082 284-21008/? E/mm-camera-img: module_imglib_msg_thread thread_id is 21008
> 09-09 21:05:00.122 284-21011/? E/mm-camera: cpp_thread_func thread_id is 21011
> 09-09 21:05:00.122 284-21011/? D/mm-camera: cpp_thread_func:61: cpp_thread entering the polling loop...
> 09-09 21:05:00.122 284-21004/? D/mm-camera: cpp_module_start_session:404, info: session 1 started.
>     c2d_module_start_session:267, info: starting session 1
> 09-09 21:05:00.122 284-21000/? E/mm-camera-sensor: module_compatile_change_chromatix_lib:293 set chromatix lib name : qtech_ar0542
> 09-09 21:05:00.122 284-21012/? D/mm-camera: c2d_thread_func:43: c2d_thread entering the polling loop...
> 09-09 21:05:00.122 284-21004/? D/mm-camera: c2d_module_start_session:339, info: session 1 started.
> 09-09 21:05:00.132 284-21000/? E/mm-camera-sensor: af_actuator_init:337 name = dw9714
>     module_sensor_start_session:761 failed
> 09-09 21:05:00.132 284-21013/? E/mm-camera: sensor_thread_func thread_id is 21013
> 09-09 21:05:00.132 284-21000/? E/mm-camera-sensor: module_sensor_start_session:767 failed
> 09-09 21:05:00.142 284-284/? I/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_interface.cpp(354): "Create Server : 0xb916f448"
>     vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_interface.cpp(354): "Create Server : 0xb8efae40"
> 09-09 21:05:00.142 284-21015/? I/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_interface.cpp(61): "Server: Waiting on new connection...\n"
> 09-09 21:05:00.142 284-21016/? E/mm-camera: mct_controller_thread_run thread_id is 21016
> 09-09 21:05:00.142 259-961/? E/mm-camera-intf: mm_camera_open:  opened, break out while loop
> 09-09 21:05:00.142 284-21014/? W/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_process.cpp(382): "No dump file present"
> 09-09 21:05:00.142 259-961/? E/QCamera2HWI: int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl_cnt =  18
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[0] =  2592 * 1944
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[1] =  2560 * 1920
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[2] =  2560 * 1440
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[3] =  2048 * 1536
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[4] =  1920 * 1080
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[5] =  1600 * 1200
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[6] =  1280 * 960
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[7] =  1280 * 768
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[8] =  1280 * 720
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[9] =  1024 * 768
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[10] =  800 * 600
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[11] =  800 * 480
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[12] =  720 * 480
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[13] =  640 * 480
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[14] =  352 * 288
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[15] =  320 * 240
>     int qcamera::QCamera2HardwareInterface::openCamera(): picture_sizes_tbl[16] =  176 * 144
> 09-09 21:05:00.142 259-961/? D/QCamera2HWI: int qcamera::QCamera2HardwareInterface::openCamera(): enable_4k2k is 0
> 09-09 21:05:00.142 259-961/? E/QCamera2HWI: int qcamera::QCamera2HardwareInterface::openCamera(): disable_1080p_preview
> 09-09 21:05:00.142 259-961/? D/mm-jpeg-intf: jpeg_open gMmCameraJpegLogLevel=1
> 09-09 21:05:00.172 814-20990/? D/dalvikvm: GC_FOR_ALLOC freed 5K, 17% free 17091K/20476K, paused 118ms, total 118ms
> 09-09 21:05:00.172 814-20990/? I/dalvikvm-heap: Grow heap (frag case) to 19.554MB for 786452-byte allocation
> 09-09 21:05:00.192 259-961/? D/mm-jpeg-intf: Max picture size 2592 x 1944, WorkBufSize = 7809024
> 09-09 21:05:00.272 252-252/? I/DEBUG:     r0 00000009  r1 c0186201  r2 bebad0f0  r3 bebad0e8
>         r4 5b8006b0  r5 5b800680  r6 5b8006e0  r7 00000036
>         r8 00000001  r9 00007205  sl 00007206  fp 00007211
>         ip 00000001  sp bebad0d8  lr 400de357  pc 400d17b4  cpsr 60000010
>         d0  00000003000000f0  d1  000000000000004a
>         d2  000000b40000003d  d3  603d4d7000000060
>         d4  006f00720064006e  d5  0072005f00640069
>         d6  0074006f0062006f  d7  0072006f0066005f
>         d8  3ff0000000000000  d9  403e000000000000
>         d10 0000000000000000  d11 0000000000000000
>         d12 0000000000000000  d13 0000000000000000
>         d14 0000000000000000  d15 0000000000000000
>         d16 3ff0000000000000  d17 0036003500340033
>         d18 002e002d002c002b  d19 0030002f002d002e
>         d20 0028002700250026  d21 0029002a002a0029
>         d22 0001000200020001  d23 0006000500040003
>         d24 0031003200320031  d25 0036003500340033
>         d26 0000000000000000  d27 0000000000000000
>         d28 001e001d001c001b  d29 0020001f001d001e
>         d30 0036003600360036  d31 0000000000000000
>         scr 60000012
> 09-09 21:05:00.282 252-252/? I/DEBUG: backtrace:
>         #00  pc 000207b4  /system/lib/libc.so (__ioctl+8)
>         #01  pc 0002d353  /system/lib/libc.so (ioctl+14)
>         #02  pc 0001d41d  /system/lib/libbinder.so (android::IPCThreadState::talkWithDriver(bool)+140)
>         #03  pc 0001d907  /system/lib/libbinder.so (android::IPCThreadState::waitForResponse(android::Parcel*, int*)+42)
>         #04  pc 0001db0f  /system/lib/libbinder.so (android::IPCThreadState::transact(int, unsigned int, android::Parcel const&, android::Parcel*, unsigned int)+118)
>         #05  pc 000197c1  /system/lib/libbinder.so (android::BpBinder::transact(unsigned int, android::Parcel const&, android::Parcel*, unsigned int)+30)
>         #06  pc 000236e9  /system/lib/libcamera_client.so
>         #07  pc 0002849b  /system/lib/libcamera_client.so (android::CameraBase<android::Camera, android::CameraTraits<android::Camera> >::connect(int, android::String16 const&, int)+142)
>         #08  pc 0001f571  /system/lib/libcamera_client.so (android::Camera::connect(int, android::String16 const&, int)+4)
>         #09  pc 000857f3  /system/lib/libandroid_runtime.so
>         #10  pc 000204cc  /system/lib/libdvm.so (dvmPlatformInvoke+112)
>         #11  pc 00051157  /system/lib/libdvm.so (dvmCallJNIMethod(unsigned int const*, JValue*, Method const*, Thread*)+398)
>         #12  pc 00029960  /system/lib/libdvm.so
>         #13  pc 00030df0  /system/lib/libdvm.so (dvmMterpStd(Thread*)+76)
>         #14  pc 0002e488  /system/lib/libdvm.so (dvmInterpret(Thread*, Method const*, JValue*)+184)
>         #15  pc 0006389d  /system/lib/libdvm.so (dvmInvokeMethod(Object*, Method const*, ArrayObject*, ArrayObject*, ClassObject*, bool)+392)
>         #16  pc 0006b7a3  /system/lib/libdvm.so
>         #17  pc 00029960  /system/lib/libdvm.so
> 09-09 21:05:00.292 252-252/? I/DEBUG:     #18  pc 00030df0  /system/lib/libdvm.so (dvmMterpStd(Thread*)+76)
>         #19  pc 0002e488  /system/lib/libdvm.so (dvmInterpret(Thread*, Method const*, JValue*)+184)
>         #20  pc 000635b9  /system/lib/libdvm.so (dvmCallMethodV(Thread*, Method const*, Object*, bool, JValue*, std::__va_list)+336)
>         #21  pc 0004cd37  /system/lib/libdvm.so
>         #22  pc 0004db63  /system/lib/libandroid_runtime.so
>         #23  pc 0004e8c7  /system/lib/libandroid_runtime.so (android::AndroidRuntime::start(char const*, char const*)+354)
>         #24  pc 0000105b  /system/bin/app_process
>         #25  pc 0000e563  /system/lib/libc.so (__libc_init+50)
>         #26  pc 00000d7c  /system/bin/app_process
>     stack:
>              bebad098  00000018  
>              bebad09c  4155cb0c  /system/lib/libdvm.so (dvmMalloc(unsigned int, int)+100)
>              bebad0a0  41f68728  [heap]
>              bebad0a4  41e098c8  
>              bebad0a8  00000000  
>              bebad0ac  00000000  
>              bebad0b0  00000000  
>              bebad0b4  00000000  
> 09-09 21:05:00.292 967-967/? D/StatusBar.NetworkController: refreshViews: Data not connected!! Set no data type icon / Roaming
> 09-09 21:05:00.292 252-252/? I/DEBUG:          bebad0b8  00000000  
>              bebad0bc  5b8006b0  
> 09-09 21:05:00.292 814-20990/? D/dalvikvm: GC_FOR_ALLOC freed 512K, 19% free 17351K/21248K, paused 115ms, total 115ms
> 09-09 21:05:00.292 252-252/? I/DEBUG:          bebad0c0  00000020  
>              bebad0c4  5b8006e0  
>              bebad0c8  00000001  
>              bebad0cc  00000001  
>              bebad0d0  00007205  
> 09-09 21:05:00.292 814-20990/? I/dalvikvm-heap: Grow heap (frag case) to 19.558MB for 524334-byte allocation
> 09-09 21:05:00.292 252-252/? I/DEBUG:          bebad0d4  00007206  
>         #00  bebad0d8  00000009  
>              ........  ........
>         #01  bebad0d8  00000009  
>              bebad0dc  bebad0e8  [stack]
> 09-09 21:05:00.302 252-252/? I/DEBUG:          bebad0e0  40178421  /system/lib/libbinder.so (android::IPCThreadState::talkWithDriver(bool)+144)
>              bebad0e4  c0186201  
>              bebad0e8  bebad0f0  [stack]
>              bebad0ec  00000018  
>         #02  bebad0f0  00000018  
>              bebad0f4  00000018  
>              bebad0f8  5b800830  
>              bebad0fc  00000100  
>              bebad100  00000000  
>              bebad104  5b800728  
>              bebad108  00000000  
>              bebad10c  5b800680  
>              bebad110  bebad250  [stack]
>              bebad114  00000000  
>              bebad118  5b8006b0  
>              bebad11c  4017890b  /system/lib/libbinder.so (android::IPCThreadState::waitForResponse(android::Parcel*, int*)+46)
> 09-09 21:05:00.322 252-252/? I/DEBUG: memory near r2:
>         bebad0d0 00007205 00007206 00000009 bebad0e8  
>         bebad0e0 40178421 c0186201 bebad0f0 00000018  
>         bebad0f0 00000018 00000018 5b800830 00000100  
>         bebad100 00000000 5b800728 00000000 5b800680  
>         bebad110 bebad250 00000000 5b8006b0 4017890b  
>         bebad120 5b8006e0 bebad144 5b80083c 4017b6ed  
> 09-09 21:05:00.322 259-961/? E/qomx_image_core: OMX_Init:94] Complete 2
> 09-09 21:05:00.322 252-252/? I/DEBUG:     bebad130 bebad220 400ff394 00000000 4017856d  
> 09-09 21:05:00.332 252-252/? I/DEBUG:     bebad140 603d4d70 00000018 00000000 00000003  
>         bebad150 00000010 00000000 00000000 000000b4  
>         bebad160 00000004 53c39709 603d4af0 00000000  
>         bebad170 5b800680 bebad250 00000000 bebad220  
>         bebad180 00000018 00000003 bebad32c 40178b13  
>         bebad190 00000003 bebad220 00000000 4017b22b  
> 09-09 21:05:00.332 259-961/? E/QCameraHWI_Mem: int qcamera::QCameraMemory::alloc(int, int, int) : No memory pool available
> 09-09 21:05:00.332 252-252/? I/DEBUG:     bebad1a0 5eca2f90 bebad1bc bebad1d8 40173cf3  
>         bebad1b0 bebad220 00000050 bebad220 0000004e  
>         bebad1c0 5ffcf740 0000004c 4041dd38 41f68738  
>     memory near r3:
>         bebad0c8 00000001 00000001 00007205 00007206  
>         bebad0d8 00000009 bebad0e8 40178421 c0186201  
>         bebad0e8 bebad0f0 00000018 00000018 00000018  
>         bebad0f8 5b800830 00000100 00000000 5b800728  
>         bebad108 00000000 5b800680 bebad250 00000000  
>         bebad118 5b8006b0 4017890b 5b8006e0 bebad144  
>         bebad128 5b80083c 4017b6ed bebad220 400ff394  
>         bebad138 00000000 4017856d 603d4d70 00000018  
>         bebad148 00000000 00000003 00000010 00000000  
>         bebad158 00000000 000000b4 00000004 53c39709  
>         bebad168 603d4af0 00000000 5b800680 bebad250  
>         bebad178 00000000 bebad220 00000018 00000003  
>         bebad188 bebad32c 40178b13 00000003 bebad220  
>         bebad198 00000000 4017b22b 5eca2f90 bebad1bc  
>         bebad1a8 bebad1d8 40173cf3 bebad220 00000050  
>         bebad1b8 bebad220 0000004e 5ffcf740 0000004c  
>     memory near r4:
>         5b800690 00000000 00000007 00000004 401846b8  
>         5b8006a0 00000000 00000000 00000007 00000004  
>         5b8006b0 00000000 5b800728 00000020 00000100  
>         5b8006c0 00000020 00000000 00000000 00000000  
>         5b8006d0 00000000 00010001 00000000 006d0061  
>         5b8006e0 00000000 5b800830 00000018 00000100  
>         5b8006f0 00000018 00000000 00000000 00000000  
>         5b800700 00000000 00010001 00000000 0065006d  
>         5b800710 ffffffff 00005137 00002789 00000204  
>         5b800720 00000000 0000010b 0000720c 80087207  
> 09-09 21:05:00.342 252-252/? I/DEBUG:     5b800730 603d4d58 603d4d3c 80087208 603d4d58  
>         5b800740 603d4d3c 00007206 000003e8 00000008  
>         5b800750 00000000 5f277158 5f277160 00007206  
>         5b800760 000003e8 00000008 00000000 5f277028  
>         5b800770 5f277030 006d0061 006e0069 00000067  
>         5b800780 006d0016 00740065 006f0068 002d0064  
>     memory near r5:
>         5b800660 40232080 5ecb13d0 40231f30 5ecb13e8  
>         5b800670 00000000 5b6fffc0 00000020 000000ab  
>         5b800680 5b800008 00005137 40184690 00000000  
>         5b800690 00000000 00000007 00000004 401846b8  
>         5b8006a0 00000000 00000000 00000007 00000004  
>         5b8006b0 00000000 5b800728 00000020 00000100  
>         5b8006c0 00000020 00000000 00000000 00000000  
>         5b8006d0 00000000 00010001 00000000 006d0061  
>         5b8006e0 00000000 5b800830 00000018 00000100  
>         5b8006f0 00000018 00000000 00000000 00000000  
>         5b800700 00000000 00010001 00000000 0065006d  
>         5b800710 ffffffff 00005137 00002789 00000204  
>         5b800720 00000000 0000010b 0000720c 80087207  
>         5b800730 603d4d58 603d4d3c 80087208 603d4d58  
>         5b800740 603d4d3c 00007206 000003e8 00000008  
>         5b800750 00000000 5f277158 5f277160 00007206  
>     memory near r6:
>         5b8006c0 00000020 00000000 00000000 00000000  
>         5b8006d0 00000000 00010001 00000000 006d0061  
>         5b8006e0 00000000 5b800830 00000018 00000100  
>         5b8006f0 00000018 00000000 00000000 00000000  
>         5b800700 00000000 00010001 00000000 0065006d  
>         5b800710 ffffffff 00005137 00002789 00000204  
>         5b800720 00000000 0000010b 0000720c 80087207  
>         5b800730 603d4d58 603d4d3c 80087208 603d4d58  
>         5b800740 603d4d3c 00007206 000003e8 00000008  
>         5b800750 00000000 5f277158 5f277160 00007206  
>         5b800760 000003e8 00000008 00000000 5f277028  
>         5b800770 5f277030 006d0061 006e0069 00000067  
>         5b800780 006d0016 00740065 006f0068 002d0064  
>         5b800790 00720074 00630061 002d0065 00720070  
>         5b8007a0 0066006f 006c0069 006e0069 00000067  
> 09-09 21:05:00.352 252-252/? I/DEBUG:     5b8007b0 0076000e 00650069 002d0077 00690068  
>     memory near r9:
>         000071e4 ffffffff ffffffff ffffffff ffffffff  
>         000071f4 ffffffff ffffffff ffffffff ffffffff  
>         00007204 ffffffff ffffffff ffffffff ffffffff  
>         00007214 ffffffff ffffffff ffffffff ffffffff  
>         00007224 ffffffff ffffffff ffffffff ffffffff  
>         00007234 ffffffff ffffffff ffffffff ffffffff  
>         00007244 ffffffff ffffffff ffffffff ffffffff  
>         00007254 ffffffff ffffffff ffffffff ffffffff  
>         00007264 ffffffff ffffffff ffffffff ffffffff  
>         00007274 ffffffff ffffffff ffffffff ffffffff  
>         00007284 ffffffff ffffffff ffffffff ffffffff  
>         00007294 ffffffff ffffffff ffffffff ffffffff  
>         000072a4 ffffffff ffffffff ffffffff ffffffff  
>         000072b4 ffffffff ffffffff ffffffff ffffffff  
>         000072c4 ffffffff ffffffff ffffffff ffffffff  
>         000072d4 ffffffff ffffffff ffffffff ffffffff  
>     memory near sl:
>         000071e4 ffffffff ffffffff ffffffff ffffffff  
>         000071f4 ffffffff ffffffff ffffffff ffffffff  
>         00007204 ffffffff ffffffff ffffffff ffffffff  
>         00007214 ffffffff ffffffff ffffffff ffffffff  
>         00007224 ffffffff ffffffff ffffffff ffffffff  
>         00007234 ffffffff ffffffff ffffffff ffffffff  
>         00007244 ffffffff ffffffff ffffffff ffffffff  
>         00007254 ffffffff ffffffff ffffffff ffffffff  
>         00007264 ffffffff ffffffff ffffffff ffffffff  
>         00007274 ffffffff ffffffff ffffffff ffffffff  
>         00007284 ffffffff ffffffff ffffffff ffffffff  
>         00007294 ffffffff ffffffff ffffffff ffffffff  
>         000072a4 ffffffff ffffffff ffffffff ffffffff  
>         000072b4 ffffffff ffffffff ffffffff ffffffff  
>         000072c4 ffffffff ffffffff ffffffff ffffffff  
>         000072d4 ffffffff ffffffff ffffffff ffffffff  
>     memory near fp:
>         000071f0 ffffffff ffffffff ffffffff ffffffff  
>         00007200 ffffffff ffffffff ffffffff ffffffff  
>         00007210 ffffffff ffffffff ffffffff ffffffff  
>         00007220 ffffffff ffffffff ffffffff ffffffff  
>         00007230 ffffffff ffffffff ffffffff ffffffff  
>         00007240 ffffffff ffffffff ffffffff ffffffff  
> 09-09 21:05:00.362 252-252/? I/DEBUG:     00007250 ffffffff ffffffff ffffffff ffffffff  
>         00007260 ffffffff ffffffff ffffffff ffffffff  
>         00007270 ffffffff ffffffff ffffffff ffffffff  
>         00007280 ffffffff ffffffff ffffffff ffffffff  
>         00007290 ffffffff ffffffff ffffffff ffffffff  
>         000072a0 ffffffff ffffffff ffffffff ffffffff  
>         000072b0 ffffffff ffffffff ffffffff ffffffff  
>         000072c0 ffffffff ffffffff ffffffff ffffffff  
>         000072d0 ffffffff ffffffff ffffffff ffffffff  
>         000072e0 ffffffff ffffffff ffffffff ffffffff  
>     memory near sp:
>         bebad0b8 00000000 5b8006b0 00000020 5b8006e0  
> 09-09 21:05:00.362 259-961/? D/QCameraHWI_Mem: static int qcamera::QCameraMemory::allocOneBuffer(qcamera::QCameraMemory::QCameraMemInfo&, int, int, bool) : ION buffer 1 with size 1048576 allocated
> 09-09 21:05:00.362 252-252/? I/DEBUG:     bebad0c8 00000001 00000001 00007205 00007206  
>         bebad0d8 00000009 bebad0e8 40178421 c0186201  
>         bebad0e8 bebad0f0 00000018 00000018 00000018  
>         bebad0f8 5b800830 00000100 00000000 5b800728  
>         bebad108 00000000 5b800680 bebad250 00000000  
>         bebad118 5b8006b0 4017890b 5b8006e0 bebad144  
>         bebad128 5b80083c 4017b6ed bebad220 400ff394  
>         bebad138 00000000 4017856d 603d4d70 00000018  
> 09-09 21:05:00.362 259-961/? D/QCameraParameters: int32_t qcamera::QCameraParameters::initDefaultParameters(): supported preview sizes: 1280x960,1280x720,864x480,800x480,768x432,720x480,640x480,576x432,480x320,384x288,352x288,320x240,240x160,176x144,160x120
> 09-09 21:05:00.362 252-252/? I/DEBUG:     bebad148 00000000 00000003 00000010 00000000  
>         bebad158 00000000 000000b4 00000004 53c39709  
>         bebad168 603d4af0 00000000 5b800680 bebad250  
>         bebad178 00000000 bebad220 00000018 00000003  
>         bebad188 bebad32c 40178b13 00000003 bebad220  
>         bebad198 00000000 4017b22b 5eca2f90 bebad1bc  
>         bebad1a8 bebad1d8 40173cf3 bebad220 00000050  
>     code around pc:
>         400d1794 ef000000 e1a0700c e3700a01 912fff1e  
>         400d17a4 e2600000 ea00754c e1a0c007 e3a07036  
>         400d17b4 ef000000 e1a0700c e3700a01 912fff1e  
>         400d17c4 e2600000 ea007544 e1a0c007 e3a07091  
>         400d17d4 ef000000 e1a0700c e3700a01 912fff1e  
>         400d17e4 e2600000 ea00753c e1a0c007 e3a07092  
>         400d17f4 ef000000 e1a0700c e3700a01 912fff1e  
>         400d1804 e2600000 ea007534 e1a0c007 e3a07037  
> 09-09 21:05:00.362 259-961/? D/QCameraParameters: int32_t qcamera::QCameraParameters::initDefaultParameters(): supported video sizes: 1920x1080,1280x960,1280x720,864x480,800x480,720x480,640x480,480x320,352x288,320x240,176x144,160x120
> 09-09 21:05:00.362 252-252/? I/DEBUG:     400d1814 ef000000 e1a0700c e3700a01 912fff1e  
>         400d1824 e2600000 ea00752c e1a0c007 e3a0708f  
> 09-09 21:05:00.362 259-961/? E/QCameraParameters: int32_t qcamera::QCameraParameters::initDefaultParameters(): KEY_PREFERRED_PREVIEW_SIZE_FOR_VIDEO j = 2
> 09-09 21:05:00.362 252-252/? I/DEBUG:     400d1834 ef000000 e1a0700c e3700a01 912fff1e  
> 09-09 21:05:00.362 259-961/? D/QCameraParameters: int32_t qcamera::QCameraParameters::initDefaultParameters(): supported pic sizes: 2592x1944,2560x1920,2560x1440,2048x1536,1920x1080,1600x1200,1280x960,1280x768,1280x720,1024x768,800x600,800x480,720x480,640x480,352x288,320x240,176x144,160x120
> 09-09 21:05:00.372 252-252/? I/DEBUG:     400d1844 e2600000 ea007524 e1a0c007 e3a0705e  
> 09-09 21:05:00.372 259-961/? D/QCameraParameters: int32_t qcamera::QCameraParameters::initDefaultParameters(): supported fps ranges: (7500,30000)
>     int32_t qcamera::QCameraParameters::setPreviewFpsRange(int, int, int, int): Updated: minFps = 7500, maxFps = 30000 , vid minFps = 7500, vid maxFps = 30000
> 09-09 21:05:00.372 259-961/? D/QCamera2HWI: int qcamera::QCamera2HardwareInterface::calcThermalLevel(qcamera::qcamera_thermal_level_enum_t, int, int, cam_fps_range_t&, msm_vfe_frame_skip_pattern&): level: 0, preview minfps 7500, preview maxfpS 30000video minfps 7500, video maxfpS 30000
> 09-09 21:05:00.372 252-252/? I/DEBUG:     400d1854 ef000000 e1a0700c e3700a01 912fff1e  
>         400d1864 e2600000 ea00751c e1a0c007 e3a07029  
> 09-09 21:05:00.372 259-961/? D/QCamera2HWI: int qcamera::QCamera2HardwareInterface::calcThermalLevel(qcamera::qcamera_thermal_level_enum_t, int, int, cam_fps_range_t&, msm_vfe_frame_skip_pattern&): Thermal level 0, FPS [7.50,30.00, 7.50,30.00], frameskip 0
> 09-09 21:05:00.372 252-252/? I/DEBUG:     400d1874 ef000000 e1a0700c e3700a01 912fff1e  
> 09-09 21:05:00.372 259-961/? D/QCameraParameters: int32_t qcamera::QCameraParameters::setPreviewFpsRange(int, int, int, int): Thermal adjusted Preview fps range 7.50,30.00, 7.50, 30.00
>     int32_t qcamera::QCameraParameters::initDefaultParameters(): supported fps rates: 8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30
> 09-09 21:05:00.372 252-252/? I/DEBUG:     400d1884 e2600000 ea007514 e1a0c007 e3a0702a  
> 09-09 21:05:00.372 259-961/? D/QCameraParameters: int32_t qcamera::QCameraParameters::setFocusMode(const char*): Setting focus mode auto
> 09-09 21:05:00.372 252-252/? I/DEBUG: code around lr:
>         400de334 fd20f7e0 f04fe001 463036ff bdf0b005  
> 09-09 21:05:00.372 259-961/? D/QCameraParameters: int32_t qcamera::QCameraParameters::setFlash(const char*): Setting Flash value off
>     int32_t qcamera::QCameraParameters::setSceneMode(const char*): Setting SceneMode auto
> 09-09 21:05:00.372 252-252/? I/DEBUG:     400de344 b503b40e 9a04ab03 1b04f853 f7f39301  
>         400de354 e8bdea2c b003400c 00004770 f2454b0c  
> 09-09 21:05:00.372 259-961/? D/QCameraParameters: int32_t qcamera::QCameraParameters::setAEBracket(const char*), EXP_BRACKETING_OFF
> 09-09 21:05:00.372 252-252/? I/DEBUG:     400de364 4a0c4101 b510447b 589cb08a 6823466a  
>         400de374 f7ff9309 9909ffe5 f1d06822 bf380001  
> 09-09 21:05:00.372 259-961/? D/QCameraParameters: int32_t qcamera::QCameraParameters::setDISValue(const char*): Setting DIS value disable
> 09-09 21:05:00.372 252-252/? I/DEBUG:     400de384 42912000 f7e4d001 b00afecb bf00bd10  
>         400de394 0001cc6c ffffff4c 0b10ec51 f021b510  
> 09-09 21:05:00.372 259-961/? D/QCameraParameters: int32_t qcamera::QCameraParameters::initDefaultParameters(): totalram = 927350784, freeram = 27131904 
> 09-09 21:05:00.372 284-21016/? E/mm-camera: mct_pipeline_process_set:command=800000e
> 09-09 21:05:00.372 252-252/? I/DEBUG:     400de3a4 ee114100 f0044a90 ec534000 ea402b10  
> 09-09 21:05:00.372 259-961/? D/QCameraParameters: int32_t qcamera::QCameraParameters::setNumOfSnapshot(): nBurstNum = 1, nExpnum = 1
> 09-09 21:05:00.372 252-252/? I/DEBUG:     400de3b4 ec430301 bd102b10 00000000 ec41b538  
>         400de3c4 46050b11 8b02ed2d 500af3c1 f021b980  
>         400de3d4 ea504000 d04d0405 0b2aeddf 1b20ee21  
>         400de3e4 429a4b30 0b11ec51 ed9fda0b e03f0b28  
> 09-09 21:05:00.372 259-961/? I/Thermal-Lib: Thermal-Lib-Client: Registraion successful for camera with handle:1
>     Thermal-Lib-Client: Registraion successful for camcorder with handle:2
> 09-09 21:05:00.372 252-252/? I/DEBUG:     400de3f4 74fff240 d00142a0 e006468c 1b01ee31  
> 09-09 21:05:00.372 259-20998/? D/QCamera2HWI: int qcamera::QCamera2HardwareInterface::enableMsgType(int32_t) (0xc0d) : mMsgEnabled = 0xc0d
> 09-09 21:05:00.382 291-336/? I/ThermalEngine: Thermal-Server: Adding thermal event listener on fd 11
> 09-09 21:05:00.382 259-21022/? I/Thermal-Lib: Thermal-Lib-Client: Client received msg camera 0
> 09-09 21:05:00.382 259-20998/? D/QCamera2HWI: int qcamera::QCamera2HardwareInterface::calcThermalLevel(qcamera::qcamera_thermal_level_enum_t, int, int, cam_fps_range_t&, msm_vfe_frame_skip_pattern&): level: 0, preview minfps 7500, preview maxfpS 30000video minfps 7500, video maxfpS 30000
>     int qcamera::QCamera2HardwareInterface::calcThermalLevel(qcamera::qcamera_thermal_level_enum_t, int, int, cam_fps_range_t&, msm_vfe_frame_skip_pattern&): Thermal level 0, FPS [7.50,30.00, 7.50,30.00], frameskip 0
> 09-09 21:05:00.382 252-252/? I/DEBUG:     400de404 f3c1e038 468c500a 18813836 73fef240  
> 09-09 21:05:00.382 284-21016/? E/mm-camera: mct_pipeline_process_set:command=800000e
> 09-09 21:05:00.382 259-21022/? I/Thermal-Lib: Thermal-Lib-Client: Client received msg camcorder 0
> 09-09 21:05:00.382 259-20998/? D/QCamera2HWI: int qcamera::QCamera2HardwareInterface::calcThermalLevel(qcamera::qcamera_thermal_level_enum_t, int, int, cam_fps_range_t&, msm_vfe_frame_skip_pattern&): level: 0, preview minfps 7500, preview maxfpS 30000video minfps 7500, video maxfpS 30000
>     int qcamera::QCamera2HardwareInterface::calcThermalLevel(qcamera::qcamera_thermal_level_enum_t, int, int, cam_fps_range_t&, msm_vfe_frame_skip_pattern&): Thermal level 0, FPS [7.50,30.00, 7.50,30.00], frameskip 0
> 09-09 21:05:00.382 284-21016/? E/mm-camera: mct_pipeline_process_set:command=800000e
> 09-09 21:05:00.382 252-252/? I/DEBUG:     400de414 dd004299 2900e013 ec53dd0a f02c2b11  
>         400de424 f42343ff ea4c0ce0 ec435301 e0212b11  
> 09-09 21:05:00.442 814-20990/? D/dalvikvm: GC_FOR_ALLOC freed 1544K, 25% free 16323K/21764K, paused 133ms, total 133ms
> 09-09 21:05:00.932 814-828/? I/Process: Sending signal. PID: 20791 SIG: 6
> 09-09 21:05:01.142 284-21014/? E/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_cam_adapter.cpp(624): "mct_stream_t* eztune::get_mct_stream(mct_pipeline_t*, eztune::mct_pipeline_get_stream_info_t*): no children"
> 09-09 21:05:01.142 284-21014/? W/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_cam_adapter.cpp(676): "Message send ignored as stream or stream state not valid"
> 09-09 21:05:01.142 284-21014/? E/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_cam_adapter.cpp(624): "mct_stream_t* eztune::get_mct_stream(mct_pipeline_t*, eztune::mct_pipeline_get_stream_info_t*): no children"
> 09-09 21:05:01.142 284-21014/? W/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_cam_adapter.cpp(676): "Message send ignored as stream or stream state not valid"
> 09-09 21:05:01.142 284-21014/? E/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_cam_adapter.cpp(624): "mct_stream_t* eztune::get_mct_stream(mct_pipeline_t*, eztune::mct_pipeline_get_stream_info_t*): no children"
> 09-09 21:05:01.142 284-21014/? W/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_cam_adapter.cpp(676): "Message send ignored as stream or stream state not valid"
> 09-09 21:05:01.142 284-21014/? E/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_cam_adapter.cpp(624): "mct_stream_t* eztune::get_mct_stream(mct_pipeline_t*, eztune::mct_pipeline_get_stream_info_t*): no children"
> 09-09 21:05:01.142 284-21014/? W/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_cam_adapter.cpp(676): "Message send ignored as stream or stream state not valid"
> 09-09 21:05:01.142 284-21014/? E/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_cam_adapter.cpp(624): "mct_stream_t* eztune::get_mct_stream(mct_pipeline_t*, eztune::mct_pipeline_get_stream_info_t*): no children"
> 09-09 21:05:01.142 284-21014/? W/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_cam_adapter.cpp(676): "Message send ignored as stream or stream state not valid"
> 09-09 21:05:01.142 284-21014/? I/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_process.cpp(400): "bool eztune::ProcessLayer::Init(eztune_server_t, int):Control Mode Init"
>     vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_interface.cpp(61): "Server: Waiting on new connection...\n"
> 09-09 21:05:01.952 814-932/? D/PowerManagerService: sendPendingNotificationsLocked mSendGoToSleepFinishedNotificationWhenReady = false
> 09-09 21:05:01.962 814-814/? W/System.err: android.provider.Settings$SettingNotFoundException: light_duration
>         at android.provider.Settings$System.getIntForUser(Settings.java:1250)
>         at android.provider.Settings$System.getInt(Settings.java:1240)
>         at com.android.server.LightsService.turnOffButtonLight(LightsService.java:96)
>         at com.android.server.LightsService$2.handleMessage(LightsService.java:301)
>         at android.os.Handler.dispatchMessage(Handler.java:102)
>         at android.os.Looper.loop(Looper.java:136)
>         at com.android.server.ServerThread.initAndLoop(SystemServer.java:1181)
>         at com.android.server.SystemServer.main(SystemServer.java:1306)
>         at java.lang.reflect.Method.invokeNative(Native Method)
> 09-09 21:05:01.962 814-814/? D/LightsService:  turnOffButtonLight   mButtonLightTimeout = 2000
> 09-09 21:05:01.972 814-814/? W/System.err:     at java.lang.reflect.Method.invoke(Method.java:515)
>         at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
>         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:602)
>         at dalvik.system.NativeStart.main(Native Method)
> 09-09 21:05:02.042 814-4498/? D/PowerManagerService: sendPendingNotificationsLocked mSendGoToSleepFinishedNotificationWhenReady = false
> 09-09 21:05:02.042 814-21024/? W/ActivityManager:   Force finishing activity com.runmyrobot.android_robot_for_phone/.activities.MainRobotActivity
> 09-09 21:05:02.042 814-21024/? D/PowerManagerService: sendPendingNotificationsLocked mSendGoToSleepFinishedNotificationWhenReady = false
> 09-09 21:05:02.052 967-967/? D/StatusBar.NetworkController: refreshViews: Data not connected!! Set no data type icon / Roaming
> 09-09 21:05:02.072 967-967/? D/LockScreen_KeyguardUpdateMonitor: received broadcast android.intent.action.TIME_TICK
> 09-09 21:05:02.072 967-967/? D/LockScreen_DMLockCodeController: canAcceptIntent - can: false
> 09-09 21:05:02.072 967-967/? D/LockScreen_CMASAlertController: canAcceptIntent - can: false
> 09-09 21:05:02.072 967-967/? D/LockScreen_KeyguardUpdateMonitor: handleTimeUpdate
> 09-09 21:05:02.072 814-1195/? I/ActivityManager: Process com.runmyrobot.android_robot_for_phone (pid 20791) has died.
> 09-09 21:05:02.082 259-8923/? W/CameraService: Disconnecting camera client 0xb77367a8 since the binder for it died (this pid 259)
> 09-09 21:05:02.082 259-20998/? D/QCamera2HWI: int qcamera::QCamera2HardwareInterface::disableMsgType(int32_t) (0xffff) : mMsgEnabled = 0x0
> 09-09 21:05:02.082 259-8923/? D/QCamera2HWI: [KPI Perf] static void qcamera::QCamera2HardwareInterface::stop_preview(camera_device*): E PROFILE_STOP_PREVIEW
> 09-09 21:05:02.082 259-20998/? D/QCameraStateMachine: int32_t qcamera::QCameraStateMachine::procEvtPreviewStoppedState(qcamera::qcamera_sm_evt_enum_t, void*): already in preview stopped state, do nothing
> 09-09 21:05:02.082 259-8923/? D/QCamera2HWI: [KPI Perf] static void qcamera::QCamera2HardwareInterface::stop_preview(camera_device*): X
>     [KPI Perf] static int qcamera::QCamera2HardwareInterface::cancel_picture(camera_device*): E PROFILE_CANCEL_PICTURE
> 09-09 21:05:02.082 259-20998/? D/QCameraStateMachine: int32_t qcamera::QCameraStateMachine::procEvtPreviewStoppedState(qcamera::qcamera_sm_evt_enum_t, void*): No ops for evt(20) in state(0)
> 09-09 21:05:02.082 259-8923/? D/QCamera2HWI: [KPI Perf] static int qcamera::QCamera2HardwareInterface::cancel_picture(camera_device*): X
> 09-09 21:05:02.082 259-8923/? I/CameraClient: Destroying camera 0
> 09-09 21:05:02.082 259-8923/? D/QCamera2HWI: [KPI Perf] static int qcamera::QCamera2HardwareInterface::close_camera_device(hw_device_t*): E
>     virtual qcamera::QCamera2HardwareInterface::~QCamera2HardwareInterface(): E
> 09-09 21:05:02.082 259-20999/? D/QCamera2HWI: static void* qcamera::QCamera2HardwareInterface::defferedWorkRoutine(void*): stop data proc
> 09-09 21:05:02.082 259-8923/? D/QCamera2HWI: int qcamera::QCamera2HardwareInterface::closeCamera(): E
> 09-09 21:05:02.082 814-1112/? I/WindowState: WIN DEATH: Window{42ab2b38 u0 SurfaceView}
> 09-09 21:05:02.082 814-1194/? I/WindowState: WIN DEATH: Window{42a81600 u0 com.runmyrobot.android_robot_for_phone/com.runmyrobot.android_robot_for_phone.activities.MainRobotActivity}
> 09-09 21:05:02.082 814-1194/? W/WindowManager: Force-removing child win Window{42ab2b38 u0 SurfaceView EXITING} from container Window{42a81600 u0 com.runmyrobot.android_robot_for_phone/com.runmyrobot.android_robot_for_phone.activities.MainRobotActivity}
> 09-09 21:05:02.092 259-8923/? E/qomx_image_core: OMX_Deinit:118] Complete
> 09-09 21:05:02.122 967-967/? E/niejiyong: WeatherService is Running
>     Weather process is Running
> 09-09 21:05:02.132 259-8923/? E/QCameraPostProc: int32_t qcamera::QCameraPostProcessor::deinit(): Jpeg closed, rc = 0, mJpegClientHandle = 3100
> 09-09 21:05:02.132 259-8923/? I/Thermal-Lib: Thermal-Lib-Client: Unregisteration is successfull for handle:1
> 09-09 21:05:02.132 291-336/? I/ThermalEngine: Thermal-Server: removing client on fd 11
> 09-09 21:05:02.132 259-8923/? I/Thermal-Lib: Thermal-Lib-Client: Unregisteration is successfull for handle:2
> 09-09 21:05:02.132 284-21016/? E/mm-camera: mct_pipeline_process_set:command=800000b
> 09-09 21:05:02.132 284-284/? I/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_interface.cpp(368): "Delete Server : 0xb916f448"
> 09-09 21:05:02.132 284-21014/? W/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/mmcam_socket_utils.cpp(310): "Received event from non socket fd"
>     vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_interface.cpp(71): "Stop message. Exiting server thread"
> 09-09 21:05:02.132 284-21014/? E/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_cam_adapter.cpp(624): "mct_stream_t* eztune::get_mct_stream(mct_pipeline_t*, eztune::mct_pipeline_get_stream_info_t*): no children"
> 09-09 21:05:02.132 284-21014/? W/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_cam_adapter.cpp(676): "Message send ignored as stream or stream state not valid"
> 09-09 21:05:02.132 284-21014/? E/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_cam_adapter.cpp(624): "mct_stream_t* eztune::get_mct_stream(mct_pipeline_t*, eztune::mct_pipeline_get_stream_info_t*): no children"
> 09-09 21:05:02.132 284-21014/? W/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_cam_adapter.cpp(676): "Message send ignored as stream or stream state not valid"
> 09-09 21:05:02.132 284-21014/? E/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_cam_adapter.cpp(624): "mct_stream_t* eztune::get_mct_stream(mct_pipeline_t*, eztune::mct_pipeline_get_stream_info_t*): no children"
> 09-09 21:05:02.132 284-21014/? W/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_cam_adapter.cpp(676): "Message send ignored as stream or stream state not valid"
> 09-09 21:05:02.142 284-284/? I/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_interface.cpp(368): "Delete Server : 0xb8efae40"
> 09-09 21:05:02.142 259-8923/? D/QCamera2HWI: int qcamera::QCamera2HardwareInterface::closeCamera(): X
>     virtual qcamera::QCamera2HardwareInterface::~QCamera2HardwareInterface(): X
> 09-09 21:05:02.142 259-20998/? D/QCameraStateMachine: static void* qcamera::QCameraStateMachine::smEvtProcRoutine(void*): X
> 09-09 21:05:02.142 284-21015/? W/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/mmcam_socket_utils.cpp(310): "Received event from non socket fd"
>     vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_interface.cpp(71): "Stop message. Exiting server thread"
> 09-09 21:05:02.142 284-21015/? E/mm-camera-eztune: vendor/qcom/proprietary/mm-camera/mm-camera2/server-tuning/tuning/eztune_process.cpp(435): "Process layer not initialized. just return"
> 09-09 21:05:02.142 259-8923/? D/QCamera2HWI: [KPI Perf] static int qcamera::QCamera2HardwareInterface::close_camera_device(hw_device_t*): X
> 09-09 21:05:02.142 284-21027/? E/mm-camera: mct_pipeline_stop_session_thread thread_id is 21027
> 09-09 21:05:02.142 284-21027/? E/mm-camera-sensor: module_sensor_stop_session:786 session 1
> 09-09 21:05:02.142 284-21028/? E/mm-camera: mct_pipeline_stop_session_thread thread_id is 21028
> 09-09 21:05:02.142 284-21029/? E/mm-camera: mct_pipeline_stop_session_thread thread_id is 21029
> 09-09 21:05:02.152 284-21030/? E/mm-camera: mct_pipeline_stop_session_thread thread_id is 21030
> 09-09 21:05:02.162 284-21031/? E/mm-camera: mct_pipeline_stop_session_thread thread_id is 21031
> 09-09 21:05:02.162 284-21031/? D/mm-camera: cpp_module_stop_session:420, info: stopping session 1 ...
> 09-09 21:05:02.162 284-21011/? D/mm-camera: cpp_thread_process_pipe_message:375, CPP_THREAD_MSG_ABORT: cpp_thread exiting..
> 09-09 21:05:02.162 284-21032/? E/mm-camera: mct_pipeline_stop_session_thread thread_id is 21032
> 09-09 21:05:02.162 284-21032/? E/hdr: Session id 0x1 stopped
> 09-09 21:05:02.162 284-21031/? D/mm-camera: cpp_module_stop_session:453, info: session 1 stopped.
>     c2d_module_stop_session:355, info: stopping session 1 ...
> 09-09 21:05:02.162 284-21012/? D/mm-camera: c2d_thread_process_pipe_message:713, C2D_THREAD_MSG_ABORT: c2d_thread exiting..
> 09-09 21:05:02.162 284-21031/? D/mm-camera: c2d_module_stop_session:399, info: session 1 stopped.
> 09-09 21:05:02.162 284-21013/? E/mm-camera: Got some events
```

**Comment by btelman96** [comment]
Probably will not be able to figure this one out. Might be just a device specific, but not sure. This happened during exiting and entering the app multiple times after establishing a connection

**Comment by btelman96** [comment]
Have not gotten this to appear on a Moto Z

**Comment by btelman96** [comment]
New ticket will open automatically if this happens and error reporting is turned on
